HHS/Head-Start-TTADP
# [TTAHUB-1570] - Standardize spacing around main content

## Description of change
Adjusted css classes to create consistent 5x padding around the main content. Also added css changes to support responsive behavior and to keep content area flush against left navigation.


## How to test
Inspect div id#appwrapper in dev tools to see that margin is the same width as the SiteNav. Reduce screen to less than 640px wide to see that padding around main content area reduces from 5x (40px) to 3x (24px). Check that padding around main content is consistent all the way around - with the exception of Recipient record tabbed pages, where padding at top of content is applied on the tabs as bottom margin. Content on the tabbed pages no longer requires left margin to keep the consistent spacing around the main content area.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1570


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
